src/madpack/madpack.py

```diff
@@ -1482,6 +1484,9 @@ def main(argv):
                 else:
                     # only need the first two digits for <= 4.3.4
                     dbver = '.'.join(map(str, dbver_split[:2]))
+            elif portid == 'cloudberry':
+                # Assume Cloudberry will stick to semantic versioning
```

Comment:
`Assume Cloudberry will stick to semantic versioning` should be `Assume Apache Cloudberry will stick to semantic versioning`